adhesive/telegram_bot.py

```diff
@@ -220,25 +219,22 @@ def format_draft_message():
 			event.client.http,
 			meta,
 			token=event.client.config['signal']['stickers']['signalstickers_api_key'],
-			# TODO use configured log levels for this i guess
-			test_mode=event.client.config['signal']['stickers'].get('signalstickers_api_test_mode', False),
 		)
 
 		async with anyio.create_task_group() as tg:
 			await tg.spawn(partial(draft_msg.edit, buttons=None))
 			await tg.spawn(processing_message.delete)
 
-		if status_code not in range(200, 300):
+		if status_code != 200:
 			await event.reply(
 				"Ruh roh. Looks like we got an error from signalstickers.com. Here's what they said: "
 				f'“{data["error"]}”'
 			)
 		else:
 			await event.reply(
-				'Yuh, I submitted your pack to signalstickers.com. It will now be reviewed by a real meat-popsicle! '
-				f'Check it out here: {data["pr_url"]}\n'
-				'If you have a GitHub account you can comment on it there, '
-				"or if you don't, you can DM [@signalstickers on Twitter](https://twitter.com/signalstickers).",
+				"Yuh, I submitted your pack to signalstickers.com. It will now be reviewed by a real meat-popsicle! "
+				"[You can check its review status here](https://signalstickers.com/contribution-status).\n"
```

Comment:
Why is the link static?

Reply:
For the moment, it is required that the user pastes its `signal.art` pack's URL. It might change in the future (e.g. add the `signal.art` url as a parameter in the URL), so I added a comment

Reply:
I would rather you fix this regression before proceeding.

Reply:
I'd rather not; I'm going on vacation soon, and I probably won't have time to implement this feature on the front-end by then. Manually merging the packs from Github is a real pain, so I'd rather merge this first, and do another PR to update the review status link when I'll be available

Reply:
Mm okay.